src/OnlineRetailPortal.Contracts/Models/Category.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Contracts
+{
+    public enum Category
```

Comment:
remove the classes which are not declared by you

Reply:
I need those classes because I am using AddProductStoreRequest and Response methods which needs those classes